### ISSUE TITLE
Tokens not found

### ISSUE THREAD

**Issue by Macmee** [servo/servo#13343]
Tokens not found
### URL:
http://reddit6.com
### Servo Version:
Servo 0.0.1-7c0dfd0
### Backtrace:
```

WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <Constellation>: No frame associated with pipeline PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }
WARNING: <ScriptThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }>: FIXME: </script> not fully implemented
WARNING: <ScriptThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(1) }>: Warning at http://reddit6.com/l.js:994:0: Using //@ to indicate sourceMappingURL pr
...
```
This report was generated by the browser.html issue reporter.

**Comment by jdm** [comment]
@Macmee Do you have any steps to reproduce this? I pushed some random buttons on reddit6.com but didn't come across anything suspicious.

**Comment by jdm** [comment]
That being said, the panic message suggests that this is a duplicate of #12799 and will be fixed by #13311.

**Comment by jdm** [comment]
This should be fixed in tomorrow's nightly. Thanks for reporting it!
